tekton/release-pipeline.yaml

```diff
@@ -85,6 +85,22 @@ spec:
         inputs:
           - name: source
             resource: source-repo
+    - name: build-base-image
+      runAfter: [build, unit-tests]
+      taskRef:
+        name: build-multiarch-base-image
```

Comment:
Where is this defined ? :sweat_smile:

Reply:
https://github.com/tektoncd/pipeline/blob/master/tekton/build-push-ma-base-image.yaml#L4